statsmodels/survey/summary_stats.py

```diff
@@ -0,0 +1,569 @@
+        stat : object
+            Object of class SurveyMean, SurveyTotal, SurveyPercentile, etc
+
+        Returns
+        -------
+        est : ndarray
+            The point estimates of the statistic, calculated on the columns
+            of data.
+        vc : square ndarray
+            The variance-covariance matrix of the estimates, obtained using
+            the (drop 1) jackknife procedure.
+        pseudo : ndarray
+            The jackknife pseudo-values.
+        """
+
+        jdata = []
+        # for each cluster
+        for c in range(self.design.nclust):
+            # get jackknife weights
+            self.w = self.design.get_rep_weights(c)
+            if index is not None:
+                # 3d array, nclust x col x len(quantiles)
+                jdata.append(self._stat(self.w, index))
+            else:
+                jdata.append(self._stat(self.w))
+        jdata = np.asarray(jdata)
+        nh = self.design.ncs[self.design.sfclust].astype(np.float64)
+        # pseudo = jdata + nh[:, None] * (np.dot(self.w, stat.data) - jdata)
+        if index is not None:
+            est = self._stat(self.design.weights, index)
+        else:
+            est = self._stat(self.design.weights)
+
+        if self.mse:
+            print('mse method')
+            jdata -= est
+        else:
+            for s in range(self.design.nstrat):
+                # get indices of all clusters within a stratum
+                ii = np.flatnonzero(self.design.sfclust == s)
+                # center the 'delete 1' statistic
+                jdata[ii, :] -= jdata[ii, :].mean(0)
+        mh = np.sqrt((nh - 1) / nh)
+        fh = np.sqrt(1 - self.design.fpc)
+        jdata = fh[:, None] * mh[:, None] * jdata
+        vc = np.dot(jdata.T, jdata)
+
+        return est, np.sqrt(np.diag(vc))
+
+
+class SurveyMean(SurveyStat):
+    """
+    Calculates the mean for each column.
+
+    Parameters
+    -------
+    design : SurveyDesign object
+    data : ndarray
+        nxp array of the data to calculate the mean on
+    method: string
+        User inputs whether to get bootstrap or jackknife SE
+
+    Attributes
+    ----------
+    data : ndarray
+        The data which to calculate the mean on
+    design :
+        Points to the SurveyDesign object
+    est : ndarray
+        The point estimates of the statistic, calculated on the columns
+        of data.
+    vc : ndarray
+        The variance-covariance of the estimates.
+    pseudo : ndarray
+        The jackknife pseudo-values.
+    """
+    def __init__(self, design, data, mse=False, replicates=None, bsn=None):
+        super().__init__(design, mse)
+        self.data = np.asarray(data)
+        if self.design.se_method == "jack":
+            self.est, self.vc = self.jack()
+            # self.vc = np.sqrt(np.diag(self.vc))
+        elif self.design.se_method == "boot":
+            self.est, self.vc = self.bootstrap(replicates, bsn)
+            self.vc = np.sqrt(self.vc)
```

Comment:
`vc` is too short and sounds like "variance components" to me.  I suggest `vcov` or `varcov` for variance or covariance matrix.  Also if taking square roots a different name should be used, like `std` or `stderr`.